Library/Formula/plplot.rb

```diff
@@ -13,201 +11,62 @@ class Plplot < Formula
     sha256 "39b18e81232ae3987018662035c49a441ddd0f0fcbe9d7534620d512b7f2910c" => :mountain_lion
   end
 
-  depends_on 'cmake' => :build
-  depends_on 'pkg-config' => :build
-  depends_on 'pango'
-  depends_on :x11 => :optional
+  option "with-java"
 
-  option 'with-java'
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pango"
+  depends_on "libtool" => :run
+  depends_on "freetype"
+  depends_on :x11 => :recommended
```

Comment:
@mikemcquaid may object to making this `:recommended`. I suspect he made this `:optional`.